src/morphosamplers/samplers/dipole_samplers/disk_sampler.py

```diff
@@ -22,15 +23,12 @@ def sample(self, obj: Dipole) -> Points:
         hexagon_x = hexagon_x[inside_circle]
         hexagon_y = hexagon_y[inside_circle]
         hexagon_z = np.zeros_like(hexagon_x)
-        hexagon_xyz = einops.rearrange([hexagon_x, hexagon_y, hexagon_z], 'xyz b -> b xyz') # (n, 3)
-        hexagon_xyz = einops.rearrange(hexagon_xyz, 'b xyz -> b 1 xyz')
-        centers = np.array(obj.center) # (m, 3)
-        centers = einops.rearrange(centers, 'b xyz -> 1 b xyz')
-        final_points = hexagon_xyz + centers
+        hexagon_xyz = einops.rearrange([hexagon_x, hexagon_y, hexagon_z], 'xyz b -> b xyz')
+        pose_sampler = PoseSampler()
+        pose_set = pose_sampler.sample(obj)
+        centers = pose_set.positions
+        orientations = pose_set.orientations
+        rotated_points = hexagon_xyz @ orientations
```

Comment:
```suggestion
        rotated_points = hexagon_xyz @ orientations
```

could we get the matrix multiplication semantics correct here too please? Matrix on the left, multiplying `(3, 1)` column vectors